includes/wsal-functions.php

```diff
@@ -10,36 +10,38 @@
 add_filter( 'wsal_load_public_sensors', 'wsal_gravityforms_extension_load_public_sensors' );
 add_action( 'wsal_togglealerts_append_content_to_toggle', 'append_content_to_toggle' );
 add_filter( 'wsal_load_on_frontend', 'wsal_gravityforms_allow_sensor_on_frontend', 10, 2 );
+add_action( 'init', 'wsal_gravityforms_init' );
 
 
 function wsal_gravityforms_extension_load_public_sensors( $value ) {
-  $value[] = 'Gravity_Forms';
-  return $value;
+	$value[] = 'Gravity_Forms';
+	return $value;
 }
 
 function wsal_gravityforms_allow_sensor_on_frontend( $default, $frontend_events ) {
-  $should_load = ( $default || ! empty( $frontend_events['gravityforms'] ) ) ? true : false;;
-  return $should_load;
+	$should_load = ( $default || ! empty( $frontend_events['gravityforms'] ) ) ? true : false;
```

Comment:
This could be simplified like this:
```suggestion
	return ( $default || ! empty( $frontend_events['gravityforms'] );
```